src/com/palmergames/bukkit/towny/war/common/townruin/TownRuinUtil.java

```diff
@@ -160,10 +157,17 @@ public static void processRuinedTownReclaimRequest(Player player, Towny plugin)
 			Bukkit.getPluginManager().callEvent(event);
 
 			TownyMessaging.sendGlobalMessage(Translation.of("msg_town_reclaimed", resident.getName(), town.getName()));
-		} catch (TownyException e) {
+		} catch (TownyException | EconomyException e) {
 			TownyMessaging.sendErrorMsg(player,e.getMessage());
-		} catch (EconomyException ex) {
-			TownyMessaging.sendErrorMsg(player,ex.getMessage());
+		}
+	}
+
+	private static void setMayor(Towny plugin, Town town, String name) {
+		try {
```

Comment:
Please re-add the TODO: on this, so that it gets dealt with later on.

Reply:
Will do.